docs/stock/advanced-annotations.md

```diff
@@ -0,0 +1,127 @@
+        }
+
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/fibonacci allow="fullscreen"></iframe>
+
+Look for the [typeOptions](https://api.highcharts.com/highstock/annotations.fibonacci.typeOptions) and [labelOptions](https://api.highcharts.com/highstock/annotations.fibonacci.labelOptions) for a specific annotation.
+
+
+Points and control points
+-------------------------
+
+Most of the advanced annotations have some control points defined by default. These control points enable users to adjust the annotation by dragging points within the chart, allowing for easy resizing and reshaping. Control points can be modified using `controlPointOptions`, within the overall annotation object, or directly in the `points` array with `controlPoint`. For certain advanced annotations, the `typeOptions` includes an additional `heightControlPoint`, which specifically controls the height. See the Tunnel example below:
+
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/tunnel allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/tunnel) to check the code.
+
+By default, all control points are set to `visible: false`. To make them visible, change the setting to `true` in the `controlPointOptions`. Read more about [controlPointOptions](https://api.highcharts.com/highstock/annotations.tunnel.controlPointOptions) and [heightControlPoint](https://api.highcharts.com/highstock/annotations.tunnel.typeOptions.heightControlPoint).
+
+List of advanced annotations with examples
+--------------------------------
+
+Note the demos with an interactive "Apply colors" button to see how various options such as `typeOptions` are applied with different annotations.
+
+### Vertical line
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/vertical-line allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/vertical-line) to check the code.
+
+### Tunnel
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/tunnel allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/tunnel) to check the code.
+
+### Time cycles
+<iframe style="width: 100%; height: 432px; border: none;" src=https://www.highcharts.com/samples/highcharts/annotations-advanced/time-cycles allow="fullscreen"></iframe>
+
+Click [here](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/annotations-advanced/time-cycles) to check the code.
```

Comment:
@pawelfus Do we need those? Users can go to JSFiddle via a button which is always displayed in our iframes

Reply:
Good point 👍

Reply:
@pawellysy